.travis.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           env: DISTRO=ubuntu EXTERNAL=ON BUILD_UG=ON
 
 after_success:
-  - test "${TRAVIS_BUILD_STAGE_NAME}" = "Deploy" || travis_terminate 0
+  - if [[ "${TRAVIS_BUILD_STAGE_NAME}" =! [dD]eploy ]]; then travis_terminate 0; fi
```

Comment:
Shouldn't be !=

Reply:
It was 'or' (`||`), now it is 'and' due to the if statement hence I had to negate the condition.

Reply:
Just the condition `=!` ? shouldn't be `!=`

Reply:
Ah....good point.

Reply:
Fixed